rosidl_generator_py/package.xml

```diff
@@ -10,13 +10,28 @@
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
   <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
+  <buildtool_export_depend>rosidl_generator_c</buildtool_export_depend>
 
+  <build_depend>rmw</build_depend>
+  <build_depend>rmw_implementation</build_depend>
+  <build_depend>rosidl_generator_c</build_depend>
```

Comment:
The change doesn't look like whats described above. I suggested to have:
* build before run before test
* withing these groups tool before non-tools

Example list:
* buildtool
* build
* buildtool_export
* build_export
* exec
* test

This order basically resembled the workflow:
* install build tools for build platform
* install build dep for target platform
* export build tools for build platform
* export build dep for target platform
* export run dep for target platform
* install test dep if necesary